app/admin/sponsor.rb

```diff
@@ -1,7 +1,20 @@
 ActiveAdmin.register Sponsor do
 
-  preserve_default_filters!
   filter :gender, as: :select, collection: Settings.lookup.gender
+  filter :branch, as: :select, collection: proc {Branch.all}
```

Comment:
Could you specify the attribute with something like
`filter :name, label: 'Branch', as: :select, collection: proc { Branch.all }`